modules/nw-egress-ips-config-object.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/assigning-egress-ips-ovn.adoc
+
+[id="nw-egress-ips-config-object_{context}"]
+== EgressIPconfig object
+As a feature of egress IP, the `reachabilityTotalTimeoutSeconds` configures the total timeout for checks that are sent by probes to egress IP nodes. The`egressIPConfig` object allows users to set the `reachabilityTotalTimeoutSeconds` `spec`. If the EgressIP node cannot be reached within this timeout, the node is declared down.
+
+You can increase this value if your network is not stable enough to handle the current default value of 1 second.
+
+The following YAML describes changing the `reachabilityTotalTimeoutSeconds` from the default 1 second probes to 5 second probes:
+
+[source,yaml]
```

Comment:
I wonder if you may need to add a bit more context to this YAML, for example: 
```suggestion
[source,yaml]
apiVersion: k8s.ovn.org/v1
kind: EgressIP
  name: networks.operator.openshift.io
spec:
```

Reply:
I'm not 100% on these suggested values, but typically YAML specs have some context like apiversion, kind, name before the spec.